### DISCUSSION TITLE
Chat for help?

### DISCUSSION THREAD

**Discussion by dschinkel** [community/community#113111] [Copilot Conversations]
### Select Topic Area

Question

### Body

I like how MidJourney has a discord workspace where we can get instant answers to questions. Forums are so static and slow.  Does ChatGPT have a realtime chat somewhere or are we stuck with this stupid slow forum for Q&As?

**Comment by Akash1134** [comment]
Hi @dschinkel ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
